src/tests/test_cases/test.py

```diff
@@ -316,10 +314,20 @@ def sdgfsdg23245777():
 
 
 @expect('D402: First line should not be the function\'s "signature"')
-def foobar():
+def foobar(foo=1, bar=1):
+    """Signature: foobar(foo=1, bar=1)."""
+
```

Comment:
This is actually an interesting case, should we also fail if the docstring contains the keyword arguments without the default value? e.g.,

```py
def foobar(foo=1, bar=1):
   """Signature: foobar(foo, bar)."""
```

Reply:
There can be lots of combinations here with args/kwargs

Reply:
docs.python presents functions with their default values, that's why I decided to check for this kind of signature, other might argue with my logic though. There's no explicit rule about what's a signature of a function in PEP257.